### DISCUSSION TITLE
3.1.47

### DISCUSSION THREAD

**Discussion by ebkr** [ebkr/r2modmanPlus#1222] [General]
## What's Changed
* Use more lenient timeouts when downloading package lists by @anttimaki in https://github.com/ebkr/r2modmanPlus/pull/1170
* Fix "try to reconnect" on splash screen by @anttimaki in https://github.com/ebkr/r2modmanPlus/pull/1171
* Add Epic Games Store support for 20 Minutes Till Dawn by @anttimaki in https://github.com/ebkr/r2modmanPlus/pull/1173
* Improve Splash screen download indicators by @anttimaki in https://github.com/ebkr/r2modmanPlus/pull/1176
* Use caching to improve performance of large profiles by @anttimaki in https://github.com/ebkr/r2modmanPlus/pull/1151
* Improve performance when uninstalling multiple mods at once by @anttimaki in https://github.com/ebkr/r2modmanPlus/pull/1155
* Filter mod list by @anttimaki in https://github.com/ebkr/r2modmanPlus/pull/1179
* Use cached getters in LocalModList by @anttimaki in https://github.com/ebkr/r2modmanPlus/pull/1180
* Create dedicated components for mod modals by @MythicManiac in https://github.com/ebkr/r2modmanPlus/pull/1177
* Fix mod disable/uninstall status indicator by @MythicManiac in https://github.com/ebkr/r2modmanPlus/pull/1182
* Improve profile code API error handling by @MythicManiac in https://github.com/ebkr/r2modmanPlus/pull/1183
* Flush local mod list before moving to profile by @anttimaki in https://github.com/ebkr/r2modmanPlus/pull/1178
* Store list of local mods with updates on VueX by @anttimaki in https://github.com/ebkr/r2modmanPlus/pull/1181
* Refactor LocalModList.getDisabledDependencies by @anttimaki in https://github.com/ebkr/r2modmanPlus/pull/1188
* Add LocalModCard by @anttimaki in https://github.com/ebkr/r2modmanPlus/pull/1189
* Clean up code related to ordering local mod list by @anttimaki in https://github.com/ebkr/r2modmanPlus/pull/1190
* Mount dependency string modal only when it's opened by @anttimaki in https://github.com/ebkr/r2modmanPlus/pull/1192
* Update bulma by @anttimaki in https://github.com/ebkr/r2modmanPlus/pull/1187
* Manager launch parameter modal by @anttimaki in https://github.com/ebkr/r2modmanPlus/pull/1195
* Improve performance of Manager.setAllModsEnabled by @anttimaki in https://github.com/ebkr/r2modmanPlus/pull/1191
* Defer mod list filtering when user types on search input by @anttimaki in https://github.com/ebkr/r2modmanPlus/pull/1198
* Explicit installers by @MythicManiac in https://github.com/ebkr/r2modmanPlus/pull/1193
* Focus on text inputs when profile management modals are opened by @anttimaki in https://github.com/ebkr/r2modmanPlus/pull/1200
* Add "Other" (GOG) support for Cult of the Lamb by @IngoHHacks in https://github.com/ebkr/r2modmanPlus/pull/1201
* Add Palworld + Voices of the Void support, initial Unreal engine support by @ethangreen-dev in https://github.com/ebkr/r2modmanPlus/pull/1079
* Format DevEnvSetup.md by @MythicManiac in https://github.com/ebkr/r2modmanPlus/pull/1207
* Update game install rule validation by @MythicManiac in https://github.com/ebkr/r2modmanPlus/pull/1210
* Replace mod exclusions URL with a constant by @MythicManiac in https://github.com/ebkr/r2modmanPlus/pull/1208
* Code cleanup for GenericProfileInstaller by @anttimaki in https://github.com/ebkr/r2modmanPlus/pull/1209
* Update all - query performance by @ebkr in https://github.com/ebkr/r2modmanPlus/pull/1203
* Trim the code when importing a profile by @imatrisciano in https://github.com/ebkr/r2modmanPlus/pull/1194
* Use cached mod methods in remaining files by @Lordmau5 in https://github.com/ebkr/r2modmanPlus/pull/1217
* Refactor LocalModList search/sort by @anttimaki in https://github.com/ebkr/r2modmanPlus/pull/1202
* Remove any-types and Mod class by @anttimaki in https://github.com/ebkr/r2modmanPlus/pull/1204
* Improve LocalModCard performance by @anttimaki in https://github.com/ebkr/r2modmanPlus/pull/1206
* Add GitHub actions workflow for making a release by @MythicManiac in https://github.com/ebkr/r2modmanPlus/pull/1218
* Updated versions and CHANGELOG by @ebkr in https://github.com/ebkr/r2modmanPlus/pull/1220
* Develop by @ebkr in https://github.com/ebkr/r2modmanPlus/pull/1221

## New Contributors
* @IngoHHacks made their first contribution in https://github.com/ebkr/r2modmanPlus/pull/1201
* @ethangreen-dev made their first contribution in https://github.com/ebkr/r2modmanPlus/pull/1079
* @imatrisciano made their first contribution in https://github.com/ebkr/r2modmanPlus/pull/1194
* @Lordmau5 made their first contribution in https://github.com/ebkr/r2modmanPlus/pull/1217

**Full Changelog**: https://github.com/ebkr/r2modmanPlus/compare/v3.1.46...v3.1.47

<hr /><em>This discussion was created from the release <a href='https://github.com/ebkr/r2modmanPlus/releases/tag/v3.1.47'>3.1.47</a>.</em>

**Comment by Decryptu** [comment]
congrats on this update, as a user with more than 100 mods, the difference with the lags when using the app is REALLY impressive. on 3.1.46 I had to wait more than 10 seconds after any interactions, now its almost instant. 👏🏻
